### ISSUE TITLE
Building JuJu images in JuJu

### ISSUE THREAD

**Issue by KyleSanderson** [fsquillace/juju#66]
As per https://github.com/fsquillace/juju/issues/65 I think it would be really useful to be able to build JuJu in JuJu. Being able to bootstrap an Arch environment without having the parent is extremely useful. My intention is to completely build this in CI and create packages. pacstrap seemed to die when mounting, or chrooting depending on the distro underneath.

**Comment by fsquillace** [comment]
Agree, the main advantage would be that an automatic JuJu build image could be done via Travis.
Unfortunately, this cannot be achieved until the sudo command will work in JuJu:
https://github.com/fsquillace/juju/issues/59
